apps/zotonic_core/src/zotonic_core_sup.erl

```diff
@@ -53,117 +51,98 @@ init([]) ->
     spawn_delayed_status(),
     z_filehandler:start_observers(),
     LogBufferSize = z_config:get(log_http_buffer_size),
-    Processes = [
-        % Ring buffer for http request logs
-        {ringbuffer_normal,
-            {ringbuffer_process, start_link, [ zotonic_http_metrics_normal, LogBufferSize ]},
-            permanent, 5000, worker, [ ringbuffer_process ]},
-
-        {ringbuffer_prio,
-            {ringbuffer_process, start_link, [ zotonic_http_metrics_prio, LogBufferSize ]},
-            permanent, 5000, worker, [ ringbuffer_process ]},
-
-        % Http request metrics handling. Accepts priority list of buffers to consume.
-        {z_stats,
-            {z_stats, start_link, [ [ zotonic_http_metrics_prio, zotonic_http_metrics_normal ] ]},
-            permanent, 5000, worker, [z_stats]},
-
-        % SMTP gen_server for sending emails
-        {z_email_server,
-            {z_email_server, start_link, []},
-            permanent, 5000, worker, [z_email_server]},
-
-        {z_file_sup,
-            {z_file_sup, start_link, []},
-            permanent, 5000, supervisor, dynamic},
-
-        % Sites supervisor, starts all enabled sites
-        {z_sites_manager_sup,
-            {z_sites_manager_sup, start_link, []},
-            permanent, 10100, supervisor, dynamic},
-
-        % File watcher, keep track of changed files for modules, templates etc.
-        {z_file_mtime,
-            {z_file_mtime, start_link, []},
-            permanent, 10100, worker, dynamic}
-    ],
+    Processes =
+        [% Ring buffer for http request logs
+         {ringbuffer_normal,
+          {ringbuffer_process, start_link, [zotonic_http_metrics_normal, LogBufferSize]},
```

Comment:
I don't like to have indents of a singe space, makes this less readable.

Is there a setting to have all indents on tab stops?  (aka 4 chars)

Reply:
Sadly, no. That comes straight up from OTP's `prettypr`. It's really hard to change. I know, because I also like 4-space indentations and I tried my best at achieving that and failed. Without improvements on OTP's `prettypr`, I doubt that `rebar3_format` will ever be available to give you that option.